build/lib/src/builder/exceptions.dart

```diff
@@ -5,9 +5,11 @@ import '../asset/id.dart';
 
 class UnexpectedOutputException implements Exception {
   final AssetId assetId;
+  final Iterable<AssetId> expectedOutputs;
```

Comment:
[nit] can we call the `expected`? I think "Outputs" should be obvious in context.

Reply:
done